test/IRGen/debug_fragment_merge.sil

```diff
@@ -3,7 +3,7 @@
 // Checking the below on 32 bit architectures would be cumbersome: each
 // fragment is 32 bits long, which changes the number of checks as well as the
 // arithmethic on the bounds of each fragment.
-// UNSUPPORTED: OS=watchos
+// REQUIRES: CPU=arm64 || CPU=x86_64
```

Comment:
I see several tests that [also check for `arm64e`](https://github.com/apple/swift/blob/da52e752c06ddb575873189dbbdaaadd3399eaf4/test/IRGen/builtins.swift#L4), not needed here?

Reply:
Good idea, added arm64e targets as well.